src/bp/core/services/dialog/decision-engine.ts

```diff
@@ -152,7 +152,7 @@ export class DecisionEngine {
       result.executeFlows = false
       event.state.session.lastMessages.push(message)
 
-      await this.stateManager.persist(event, true)
+      await this.stateManager.persist(event, false)
```

Comment:
I don't remember why it was true, can it have an impact ? And if it's changed to false, then the parameter isn't required, since it's the only place where it was "true"

Reply:
Yeah I'm not sure either, the fact that it was true (meaning ignoreContext) was causing me issues. So I'm really not sure why we'd want to ignore save context.